album-recommendation-java/src/main/application/services.xml

```diff
@@ -2,80 +2,83 @@
-            <search> tells the container to answers queries and serve results for those queries.
-            Inside the <search /> cluster you can configure chains of "searchers" -
-            Java components processing the query and/or result.
-
-            See:
-             - Reference: https://docs.vespa.ai/en/search-api.html
-             - Searchers: https://docs.vespa.ai/en/searcher-development.html
-        -->
-        <search>
-            <chain id="metalchain" inherits="vespa">
-                <searcher id="ai.vespa.example.album.MetalSearcher" bundle="albums">
-                    <config name="ai.vespa.example.album.metal-names">
-                        <metalWords>
-                            <item>hetfield</item>
-                            <item>metallica</item>
-                            <item>pantera</item>
-                        </metalWords>
-                    </config>
-                </searcher>
-            </chain>
-        </search>
+    <document-api/>
 
-        <!--
-            <nodes> specifies the nodes that should run this cluster, and through the <resources>
-            subtag, the resources on those nodes. You can also specify ranges of nodes and resources
-            to activate autoscaling.
+    <!--
+        <search> tells the container to answers queries and serve results for those queries.
+        Inside the <search /> cluster you can configure chains of "searchers" -
+        Java components processing the query and/or result.
 
-            See:
-             - Reference: https://cloud.vespa.ai/en/reference/services.html
-        -->
-        <nodes>
-            <node hostalias="node1" />
-        </nodes>
-    </container>
+        See:
+         - Reference: https://docs.vespa.ai/en/search-api.html
+         - Searchers: https://docs.vespa.ai/en/searcher-development.html
+    -->
+    <search>
+      <chain id="default" inherits="vespa">
+        <searcher id="ai.vespa.example.album.SetFilterSearcher" bundle="albums"/>
+      </chain>
+      <chain id="metalchain" inherits="vespa">
+        <searcher id="ai.vespa.example.album.MetalSearcher" bundle="albums">
+          <config name="ai.vespa.example.album.metal-names">
+            <metalWords>
+              <item>hetfield</item>
+              <item>metallica</item>
+              <item>pantera</item>
+            </metalWords>
+          </config>
+        </searcher>
+      </chain>
+    </search>
 
     <!--
-        <content/> content clusters store application data, maintain indexes and executes the
-        distributed parts of a query.
+        <nodes> specifies the nodes that should run this cluster, and through the <resources>
+        subtag, the resources on those nodes. You can also specify ranges of nodes and resources
+        to activate autoscaling.
 
         See:
-          - Reference: https://docs.vespa.ai/en/reference/services-content.html
+         - Reference: https://cloud.vespa.ai/en/reference/services.html
```

Comment:
not sure if referring to the cloud site is better here. there is an album-recommendation-java i the vespa-cloud dir

Reply:
I have no idea what happened